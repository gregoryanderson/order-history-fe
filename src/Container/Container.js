import React from 'react';
import Order from '../Order/Order';
import './Container.css';

const Container = ({orders, deleteOrder}) => {
    const orderCards = orders.map(order => {
        return (
        <Order 
            img={order.img}
            name={order.name}
            description={order.description}
            price={order.price}
            id={order.id}
            key={order.id}
            deleteOrder={deleteOrder}
        />
        )
    })

    return (
        <section>
            {orderCards}
        </section>
    )
}

export default Container;