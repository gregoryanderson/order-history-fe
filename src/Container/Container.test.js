import React from 'react';
import { shallow } from 'enzyme';
import Container from './Container';


describe('Container', () => {
    it('should match the snapshot affliated with it', () => {
        const orders = [{  
            img:'https://www.medicalnewstoday.com/content//images/articles/322/322868/golden-retriever-puppy.jpg',
            name:"order name",
            description:"this in a nice order",
            price:500,
            id:4,
            key:4,
            deleteOrder: jest.fn()
        }]
        
        
        const wrapper = shallow(
            <Container 
            orders={orders} 
            deleteOrder={jest.fn()} 
          />
        )
    
        expect(wrapper).toMatchSnapshot()
    })
})
