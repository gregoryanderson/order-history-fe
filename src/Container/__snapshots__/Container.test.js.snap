// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot affliated with it 1`] = `ShallowWrapper {}`;
